packages/core/src/components/ThemeProvider.tsx

```diff
@@ -10,6 +10,11 @@ const theme = extendTheme({
     },
   },
   components: {
+    Modal: {
+      dialog: {
+        bg: 'var(--background-color-primary)',
+      },
+    },
```

Comment:
this is not actually a problem. I thought it was a modal at first. should we just use charka defaults? @Teddarific

Reply:
Yeah, just I'd just delete this